### PR TITLE
Migrate ApplicationAccess from couch to sql (1 of 3)

### PR DESCRIPTION
**Pull request by orangejenny**
https://dimagi-dev.atlassian.net/browse/ICDS-1144

##### SUMMARY
See [docs](https://github.com/dimagi/commcare-hq/blob/master/docs/couch_to_sql_models.rst#practical-steps) for background.

### CODE REVIEW

**Review by kaapstorm — APPROVED** [review]

File: corehq/apps/cloudcare/migrations/0001_initial.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.26 on 2019-12-12 09:59
+from __future__ import unicode_literals
```
Comment (esoergel) [inline corehq/apps/cloudcare/migrations/0001_initial.py]: nit: `__future__` imports are no longer necessary

**Review by esoergel — APPROVED** [review]
Guessing there are few enough of these that you're not concerned about running the migration twice?

**Comment by orangejenny** [comment]
@esoergel What do you mean? I'm not intending to run the migration twice, do you see an error?

**Comment by esoergel** [comment]
Nevermind - I'm incorrect.  I was expecting the idempotence to happen in this command, but I see that it's in the followup migration that runs the command (PR 3).  Carry on!

**Review by millerdev — APPROVED** [review]

**Pull request merged**